Dockerfile

```diff
@@ -1,23 +1,23 @@
-FROM node:alpine
+FROM node:6-alpine
 
-ENV appdir /usr/local/app
+ENV APPDIR /usr/local/app
 
-RUN mkdir -p $appdir
-WORKDIR  $appdir
-COPY . $appdir
+RUN mkdir -p "$APPDIR"
+WORKDIR $APPDIR
+COPY . $APPDIR
 RUN npm install
 
 RUN mkdir -p /verdaccio/storage /verdaccio/conf
 ADD conf/docker.yaml /verdaccio/conf/config.yaml
```

Comment:
According this, COPY is better than ADD . Are you agreed ? https://www.codacy.com/app/juanpicado/verdaccio/issues?&filters=W3siaWQiOiJMYW5ndWFnZSIsInZhbHVlcyI6WyJEb2NrZXJmaWxlIl19LHsiaWQiOiJDYXRlZ29yeSIsInZhbHVlcyI6W251bGxdfSx7ImlkIjoiTGV2ZWwiLCJ2YWx1ZXMiOltudWxsXX0seyJpZCI6IlBhdHRlcm4iLCJ2YWx1ZXMiOltudWxsXX0seyJpZCI6IkF1dGhvciIsInZhbHVlcyI6W251bGxdfV0=